dev/TOC.md

```diff
@@ -349,6 +349,7 @@
     - [Destroy a TiDB cluster](tidb-in-kubernetes/maintain/destroy-tidb-cluster.md)
     - [Maintain a Hosting Kubernetes Node](tidb-in-kubernetes/maintain/kubernetes-node.md)
     - [Backup and Restore](tidb-in-kubernetes/maintain/backup-and-restore.md)
+    - [Restore data with TiDB Lightning](tidb-in-kubernetes/maintain/lightning.md)
```

Comment:
In `TOC.md`, we use the headline style of capitalization. Please keep it consistent.

```suggestion
    - [Restore Data with TiDB Lightning](tidb-in-kubernetes/maintain/lightning.md)
```